package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "ibmcloud-appid",
-  "version": "6.1.0",
+  "version": "6.1.1",
   "description": "Node.js SDK for the IBM Cloud App ID service",
   "main": "lib/appid-sdk.js",
   "scripts": {
     "apiapp": "node samples/api-app-sample-server.js",
     "webapp": "node samples/web-app-sample-server.js",
     "test": "./node_modules/.bin/_mocha",
-    "test_nyc": "./node_modules/.bin/nyc ./node_modules/.bin/_mocha --timeout 30000 --exit"
```

Comment:
The following seems to be newer:
```
    "test_nyc": "./node_modules/.bin/nyc ./node_modules/.bin/_mocha --timeout 30000 --exit"
```
Please verify. Check if these build targets are mentioned in the doc.